src/content/resources/architectural-overview.md

```diff
@@ -48,6 +59,7 @@ designed to be optional and replaceable.
 The PNG diagrams in this document were created using draw.io. The draw.io
 metadata is embedded in the PNG file itself, so you can open the PNG directly
 from draw.io to edit the individual components.
+(sz: Not true, at least not for the web arch png.)
```

Comment:
Just flagging this to make sure it gets removed before release.

Reply:
Thanks! I did miss this. Anything between `%comment%` and `%endcomment%` is stripped out of the generated page. It's now fixed. :)